sharq_project/models/__init__.py

```diff
@@ -1,3 +1,5 @@
 # -*- coding: utf-8 -*-
 
-from . import models
\ No newline at end of file
+from . import models
```

Comment:
@Buniadi why the *.pyc files are commented from your side, add to ignore or exclude file